### DISCUSSION TITLE
allfontsgen does only work by (low) chance

### DISCUSSION THREAD

**Discussion by verymilan** [ONLYOFFICE/DocumentServer#2187] [General]
Hi there, I wasn't sure where to put this so I'll try here first:

**tl;dr:**
* `allfontsgen` built and run in an LXC Debian 11 container no longer outputs to the fonts directory
* `allfontsgen` in oo directory copied over from docker `out/` dir using Ubuntu 22.04 works one time only

There must be something I am not understanding correctly... please help. :) 
****
I have compiled the server on Debian 11, trying to update an existing installation (replaced the directory at `/var/www/onlyoffice`, copied new configs to `/etc/onlyoffice`). I then noticed that the server was now unable to find anything unter `/fonts` – and indeed, the directory was empty.

Running `server/tools/allfontsgen` as in the docs had no effect anymore. Since there is no way to make that command output anything, I have no idea what could be wrong.  Copying over the old fonts dir left me with incorrectly displayed fonts in the editor.

I then copied a different compiled OnlyOffice into this installation. This one was not directly built in the affected Linux Container, but with Docker – using the official example, except with `python3` only and `ubuntu:22.04`. Here i removed the already existing fonts directory and ... and `allfontsgen`worked! ... **But only one single time!** – after that, i was left with the same problem: no output, nothing.  

Douplechecked stuff:
* permissions are correct
* trying as root not helping
* removing `/sdkjs/common/AllFonts.js` not helping
* no `dmesg` output on host, no journal output in the container
* if the documentserver is running or not makes no difference

Too scared to doublecheck:
* trying to reproduce on the prod host using the built from Docker
